{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "name": "NLP.ipynb",
      "provenance": [],
      "collapsed_sections": [],
      "authorship_tag": "ABX9TyMFuPHdAqZrPnVjyc87YoPc",
      "include_colab_link": true
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    },
    "accelerator": "GPU"
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/Eitams/NLP_Exercise/blob/main/NLP.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 1,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "7U3F9F0g9biy",
        "outputId": "1512a7f1-5431-4353-a8d7-83d4bc16d5e3"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/content\n"
          ]
        }
      ],
      "source": [
        "!pwd"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "from google.colab import drive\n",
        "drive.mount('/content/drive')"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "U0fI1jy8Avk3",
        "outputId": "ef51beb8-a590-4364-cb98-05f89c29b3df"
      },
      "execution_count": 7,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Mounted at /content/drive\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "import os\n",
        "os.chdir('/content/drive/MyDrive/NLP')"
      ],
      "metadata": {
        "id": "jxaMUqSdBcNf"
      },
      "execution_count": 9,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "!pwd"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "UmTenw4KB_F5",
        "outputId": "d44db127-784b-44d8-a10c-e932801dfc96"
      },
      "execution_count": 10,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/content/drive/MyDrive/NLP\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!git clone https://github.com/Eitams/NLP_Exercise.git"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "lgAKd0IE_iOT",
        "outputId": "54dd51c9-db6b-423b-99a6-5739e29f0548"
      },
      "execution_count": 13,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Cloning into 'NLP_Exercise'...\n",
            "remote: Enumerating objects: 3, done.\u001b[K\n",
            "remote: Counting objects: 100% (3/3), done.\u001b[K\n",
            "remote: Compressing objects: 100% (2/2), done.\u001b[K\n",
            "remote: Total 3 (delta 0), reused 0 (delta 0), pack-reused 0\u001b[K\n",
            "Unpacking objects: 100% (3/3), done.\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#!pip install wikipedia-api"
      ],
      "metadata": {
        "id": "GJ4f0wpvF_eo"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "import wikipediaapi\n",
        "import pandas as pd\n",
        "\n",
        "pd.set_option('display.max_columns', None)\n",
        "#pd.set_option('display.expand_frame_repr', False)\n",
        "pd.set_option('max_colwidth', None)"
      ],
      "metadata": {
        "id": "hHQoyQseJIqr"
      },
      "execution_count": 16,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "# Get members for a category together with a short description\n",
        "def members_collector(category):\n",
        "      mdict = {}\n",
        "      categorymembers = category.categorymembers\n",
        "      for c in categorymembers.values():\n",
        "        if c.ns == 0:                                                            # Exclude categories within the category\n",
        "          mdict[c.title] = c.summary\n",
        "      return mdict"
      ],
      "metadata": {
        "id": "26Q170fkJmFW"
      },
      "execution_count": 17,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "# Define language and page\n",
        "wiki = wikipediaapi.Wikipedia('en')"
      ],
      "metadata": {
        "id": "r2kmgu1_J-WB"
      },
      "execution_count": 18,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "# Let's collect spice\n",
        "viruses = wiki.page(\"Category:Viruses\")\n",
        "viruses_dict = members_collector(viruses)"
      ],
      "metadata": {
        "id": "68QFIixIJ6kD"
      },
      "execution_count": 19,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "for i in viruses_dict:\n",
        "  print(i)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "1qXse1vRJQTm",
        "outputId": "9eb1fe79-ff18-4c69-e5f7-4dd0fce24c83"
      },
      "execution_count": 20,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Virus\n",
            "Introduction to viruses\n",
            "2021 Nipah virus outbreak in Kerala\n",
            "Alsuviricetes\n",
            "Arbovirus\n",
            "Archaeal virus\n",
            "Astrovirology\n",
            "Bacteriophage Mu\n",
            "Baltimore classification\n",
            "Botourmiaviridae\n",
            "Broad-spectrum antiviral drug\n",
            "Chimera (virus)\n",
            "Contagium vivum fluidum\n",
            "Riccardo Cortese\n",
            "Cossaviricota\n",
            "Cressdnaviricota\n",
            "Cross-species transmission\n",
            "Disease X\n",
            "Duplodnaviria\n",
            "Duplornaviricota\n",
            "Durnavirales\n",
            "Emergent virus\n",
            "Epidemiology of Hepatitis D\n",
            "Genetically modified vaccine\n",
            "Genetically modified virus\n",
            "Ghabrivirales\n",
            "Hepelivirales\n",
            "Horizontal Environmental Genetic Alteration Agents\n",
            "Human virome\n",
            "Human viruses in water\n",
            "Kitrinoviricota\n",
            "Lefavirales\n",
            "Lenarviricota\n",
            "Marine viruses\n",
            "Martellivirales\n",
            "Mosaic virus\n",
            "Naldaviricetes\n",
            "Neurotropic virus\n",
            "Nodamuvirales\n",
            "Non-cellular life\n",
            "Novel virus\n",
            "Nucleocytoviricota\n",
            "Oncovirus\n",
            "Orphan virus\n",
            "Orthornavirae\n",
            "Papovaviricetes\n",
            "Pimascovirales\n",
            "Pisoniviricetes\n",
            "Pisuviricota\n",
            "Pleolipoviridae\n",
            "Pokkesviricetes\n",
            "Preplasmiviricota\n",
            "Reporter virus particles\n",
            "Revtraviricetes\n",
            "Ribozyviria\n",
            "Robovirus\n",
            "Ruhugu virus\n",
            "Rustrela virus\n",
            "Shotokuvirae\n",
            "Sobelivirales\n",
            "Spillover infection\n",
            "Stelpaviricetes\n",
            "Tectiliviricetes\n",
            "Tibovirus\n",
            "Tolivirales\n",
            "Viral replication\n",
            "Viriome\n",
            "Virology\n",
            "Virosphere\n",
            "Social history of viruses\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# Create dataframe\n",
        "viruses_df = pd.DataFrame([viruses_dict.keys(), viruses_dict.values()]).T\n",
        "viruses_df.columns = ['Name', 'Description']\n",
        "viruses_df = viruses_df.style.set_properties(**{'text-align': 'left'}).set_table_styles([dict(selector='th', props=[('text-align', 'left')])])\n",
        "viruses_df"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "id": "uke3batfKou6",
        "outputId": "c2becc9f-7b8d-400e-f549-cdd17c7e8102"
      },
      "execution_count": 21,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "<pandas.io.formats.style.Styler at 0x7f8df6a3af10>"
            ],
            "text/html": [
              "<style type=\"text/css\">\n",
              "#T_b4c57_ th {\n",
              "  text-align: left;\n",
              "}\n",
              "#T_b4c57_row0_col0, #T_b4c57_row0_col1, #T_b4c57_row1_col0, #T_b4c57_row1_col1, #T_b4c57_row2_col0, #T_b4c57_row2_col1, #T_b4c57_row3_col0, #T_b4c57_row3_col1, #T_b4c57_row4_col0, #T_b4c57_row4_col1, #T_b4c57_row5_col0, #T_b4c57_row5_col1, #T_b4c57_row6_col0, #T_b4c57_row6_col1, #T_b4c57_row7_col0, #T_b4c57_row7_col1, #T_b4c57_row8_col0, #T_b4c57_row8_col1, #T_b4c57_row9_col0, #T_b4c57_row9_col1, #T_b4c57_row10_col0, #T_b4c57_row10_col1, #T_b4c57_row11_col0, #T_b4c57_row11_col1, #T_b4c57_row12_col0, #T_b4c57_row12_col1, #T_b4c57_row13_col0, #T_b4c57_row13_col1, #T_b4c57_row14_col0, #T_b4c57_row14_col1, #T_b4c57_row15_col0, #T_b4c57_row15_col1, #T_b4c57_row16_col0, #T_b4c57_row16_col1, #T_b4c57_row17_col0, #T_b4c57_row17_col1, #T_b4c57_row18_col0, #T_b4c57_row18_col1, #T_b4c57_row19_col0, #T_b4c57_row19_col1, #T_b4c57_row20_col0, #T_b4c57_row20_col1, #T_b4c57_row21_col0, #T_b4c57_row21_col1, #T_b4c57_row22_col0, #T_b4c57_row22_col1, #T_b4c57_row23_col0, #T_b4c57_row23_col1, #T_b4c57_row24_col0, #T_b4c57_row24_col1, #T_b4c57_row25_col0, #T_b4c57_row25_col1, #T_b4c57_row26_col0, #T_b4c57_row26_col1, #T_b4c57_row27_col0, #T_b4c57_row27_col1, #T_b4c57_row28_col0, #T_b4c57_row28_col1, #T_b4c57_row29_col0, #T_b4c57_row29_col1, #T_b4c57_row30_col0, #T_b4c57_row30_col1, #T_b4c57_row31_col0, #T_b4c57_row31_col1, #T_b4c57_row32_col0, #T_b4c57_row32_col1, #T_b4c57_row33_col0, #T_b4c57_row33_col1, #T_b4c57_row34_col0, #T_b4c57_row34_col1, #T_b4c57_row35_col0, #T_b4c57_row35_col1, #T_b4c57_row36_col0, #T_b4c57_row36_col1, #T_b4c57_row37_col0, #T_b4c57_row37_col1, #T_b4c57_row38_col0, #T_b4c57_row38_col1, #T_b4c57_row39_col0, #T_b4c57_row39_col1, #T_b4c57_row40_col0, #T_b4c57_row40_col1, #T_b4c57_row41_col0, #T_b4c57_row41_col1, #T_b4c57_row42_col0, #T_b4c57_row42_col1, #T_b4c57_row43_col0, #T_b4c57_row43_col1, #T_b4c57_row44_col0, #T_b4c57_row44_col1, #T_b4c57_row45_col0, #T_b4c57_row45_col1, #T_b4c57_row46_col0, #T_b4c57_row46_col1, #T_b4c57_row47_col0, #T_b4c57_row47_col1, #T_b4c57_row48_col0, #T_b4c57_row48_col1, #T_b4c57_row49_col0, #T_b4c57_row49_col1, #T_b4c57_row50_col0, #T_b4c57_row50_col1, #T_b4c57_row51_col0, #T_b4c57_row51_col1, #T_b4c57_row52_col0, #T_b4c57_row52_col1, #T_b4c57_row53_col0, #T_b4c57_row53_col1, #T_b4c57_row54_col0, #T_b4c57_row54_col1, #T_b4c57_row55_col0, #T_b4c57_row55_col1, #T_b4c57_row56_col0, #T_b4c57_row56_col1, #T_b4c57_row57_col0, #T_b4c57_row57_col1, #T_b4c57_row58_col0, #T_b4c57_row58_col1, #T_b4c57_row59_col0, #T_b4c57_row59_col1, #T_b4c57_row60_col0, #T_b4c57_row60_col1, #T_b4c57_row61_col0, #T_b4c57_row61_col1, #T_b4c57_row62_col0, #T_b4c57_row62_col1, #T_b4c57_row63_col0, #T_b4c57_row63_col1, #T_b4c57_row64_col0, #T_b4c57_row64_col1, #T_b4c57_row65_col0, #T_b4c57_row65_col1, #T_b4c57_row66_col0, #T_b4c57_row66_col1, #T_b4c57_row67_col0, #T_b4c57_row67_col1, #T_b4c57_row68_col0, #T_b4c57_row68_col1, #T_b4c57_row69_col0, #T_b4c57_row69_col1 {\n",
              "  text-align: left;\n",
              "}\n",
              "</style>\n",
              "<table id=\"T_b4c57_\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr>\n",
              "      <th class=\"blank level0\" >&nbsp;</th>\n",
              "      <th class=\"col_heading level0 col0\" >Name</th>\n",
              "      <th class=\"col_heading level0 col1\" >Description</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row0\" class=\"row_heading level0 row0\" >0</th>\n",
              "      <td id=\"T_b4c57_row0_col0\" class=\"data row0 col0\" >Virus</td>\n",
              "      <td id=\"T_b4c57_row0_col1\" class=\"data row0 col1\" >A virus is a submicroscopic infectious agent that replicates only inside the living cells of an organism. Viruses infect all life forms, from animals and plants to microorganisms, including bacteria and archaea.\n",
              "Since Dmitri Ivanovsky's 1892 article describing a non-bacterial pathogen infecting tobacco plants and the discovery of the tobacco mosaic virus by Martinus Beijerinck in 1898, more than 9,000 virus species have been described in detail of the millions of types of viruses in the environment. Viruses are found in almost every ecosystem on Earth and are the most numerous type of biological entity. The study of viruses is known as virology, a subspeciality of microbiology.  \n",
              "When infected, a host cell is often forced to rapidly produce thousands of copies of the original virus. When not inside an infected cell or in the process of infecting a cell, viruses exist in the form of independent particles, or virions, consisting of (i) the genetic material, i.e., long molecules of DNA or RNA that encode the structure of the proteins by which the virus acts; (ii) a protein coat, the capsid, which surrounds and protects the genetic material; and in some cases (iii) an outside envelope of lipids. The shapes of these virus particles range from simple helical and icosahedral forms to more complex structures. Most virus species have virions too small to be seen with an optical microscope and are one-hundredth the size of most bacteria.\n",
              "The origins of viruses in the evolutionary history of life are unclear: some may have evolved from plasmids—pieces of DNA that can move between cells—while others may have evolved from bacteria. In evolution, viruses are an important means of horizontal gene transfer, which increases genetic diversity in a way analogous to sexual reproduction. Viruses are considered by some biologists to be a life form, because they carry genetic material, reproduce, and evolve through natural selection, although they lack the key characteristics, such as cell structure, that are generally considered necessary criteria for defining life. Because they possess some but not all such qualities, viruses have been described as \"organisms at the edge of life\", and as replicators.Viruses spread in many ways. One transmission pathway is through disease-bearing organisms known as vectors: for example, viruses are often transmitted from plant to plant by insects that feed on plant sap, such as aphids; and viruses in animals can be carried by blood-sucking insects. Many viruses, including influenza viruses, SARS-CoV-2, chickenpox, smallpox, and measles, spread in the air by coughing and sneezing. Norovirus and rotavirus, common causes of viral gastroenteritis, are transmitted by the faecal–oral route, passed by hand-to-mouth contact or in food or water. The infectious dose of norovirus required to produce infection in humans is fewer than 100 particles. HIV is one of several viruses transmitted through sexual contact and by exposure to infected blood. The variety of host cells that a virus can infect is called its \"host range\". This can be narrow, meaning a virus is capable of infecting few species, or broad, meaning it is capable of infecting many.Viral infections in animals provoke an immune response that usually eliminates the infecting virus. Immune responses can also be produced by vaccines, which confer an artificially acquired immunity to the specific viral infection. Some viruses, including those that cause HIV/AIDS, HPV infection, and viral hepatitis, evade these immune responses and result in chronic infections. Several classes of antiviral drugs have been developed.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row1\" class=\"row_heading level0 row1\" >1</th>\n",
              "      <td id=\"T_b4c57_row1_col0\" class=\"data row1 col0\" >Introduction to viruses</td>\n",
              "      <td id=\"T_b4c57_row1_col1\" class=\"data row1 col1\" >A virus is a tiny infectious agent that reproduces inside the cells of living hosts. When infected, the host cell is forced to rapidly produce thousands of identical copies of the original virus. Unlike most living things, viruses do not have cells that divide; new viruses assemble in the infected host cell. But unlike simpler infectious agents like prions, they contain genes, which allow them to mutate and evolve. Over 4,800 species of viruses have been described in detail out of the millions in the environment. Their origin is unclear: some may have evolved from plasmids—pieces of DNA that can move between cells—while others may have evolved from bacteria.\n",
              "Viruses are made of either two or three parts. All include genes. These genes contain the encoded biological information of the virus and are built from either DNA or RNA. All viruses are also covered with a protein coat to protect the genes. Some viruses may also have an envelope of fat-like substance that covers the protein coat, and makes them vulnerable to soap. A virus with this \"viral envelope\" uses it—along with specific receptors—to enter a new host cell. Viruses vary in shape from the simple helical and icosahedral to more complex structures. Viruses range in size from 20 to 300 nanometres; it would take 33,000 to 500,000 of them, side by side, to stretch to 1 centimetre (0.4 in).\n",
              "Viruses spread in many ways. Although many are very specific about which host species or tissue they attack, each species of virus relies on a particular method to copy itself. Plant viruses are often spread from plant to plant by insects and other organisms, known as vectors. Some viruses of humans and other animals are spread by exposure to infected bodily fluids. Viruses such as influenza are spread through the air by droplets of moisture when people cough or sneeze. Viruses such as norovirus are transmitted by the faecal–oral route, which involves the contamination of hands, food and water. Rotavirus is often spread by direct contact with infected children. The human immunodeficiency virus, HIV, is transmitted by bodily fluids transferred during sex. Others, such as the dengue virus, are spread by blood-sucking insects.\n",
              "Viruses, especially those made of RNA, can mutate rapidly to give rise to new types. Hosts may have little protection against such new forms. Influenza virus, for example, changes often, so a new vaccine is needed each year. Major changes can cause pandemics, as in the 2009 swine influenza that spread to most countries. Often, these mutations take place when the virus has first infected other animal hosts. Some examples of such \"zoonotic\" diseases include coronavirus in bats, and influenza in pigs and birds, before those viruses were transferred to humans.\n",
              "Viral infections can cause disease in humans, animals and plants. In healthy humans and animals, infections are usually eliminated by the immune system, which can provide lifetime immunity to the host for that virus. Antibiotics, which work against bacteria, have no impact, but antiviral drugs can treat life-threatening infections. Those vaccines that produce lifelong immunity can prevent some infections.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row2\" class=\"row_heading level0 row2\" >2</th>\n",
              "      <td id=\"T_b4c57_row2_col0\" class=\"data row2 col0\" >2021 Nipah virus outbreak in Kerala</td>\n",
              "      <td id=\"T_b4c57_row2_col1\" class=\"data row2 col1\" >The 2021 Kerala Nipah virus outbreak was an outbreak of the zoonotic Nipah virus in Kerala. The outbreak was localized in the Pazhur village located within the Chathamangalam Gram panchayat in the Kozhikode district of Kerala, and it claimed one life. The outbreak was contained and declared over on 5 September 2021. This was the third Nipah Virus outbreak in Kerala, the earlier being in 2018 and 2019. The Central Government has rushed a team of the National Centre for Disease Control to the state, which will provide technical support. Relatives of the dead boy and all those involved in his treatment have been put under quarantine.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row3\" class=\"row_heading level0 row3\" >3</th>\n",
              "      <td id=\"T_b4c57_row3_col0\" class=\"data row3 col0\" >Alsuviricetes</td>\n",
              "      <td id=\"T_b4c57_row3_col1\" class=\"data row3 col1\" >Alsuviricetes is a class of positive-strand RNA viruses which infect eukaryotes. The name of the group is a syllabic abbreviation of \"alpha supergroup\" with the suffix -viricetes indicating a virus class.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row4\" class=\"row_heading level0 row4\" >4</th>\n",
              "      <td id=\"T_b4c57_row4_col0\" class=\"data row4 col0\" >Arbovirus</td>\n",
              "      <td id=\"T_b4c57_row4_col1\" class=\"data row4 col1\" >Arbovirus is an informal name for any virus that is transmitted by arthropod vectors. The term arbovirus is a portmanteau word (arthropod-borne virus). Tibovirus (tick-borne virus) is sometimes used to more specifically describe viruses transmitted by ticks, a superorder within the arthropods. Arboviruses can affect both animals (including humans) and plants. In humans, symptoms of arbovirus infection generally occur 3–15 days after exposure to the virus and last three or four days. The most common clinical features of infection are fever, headache, and malaise, but encephalitis and viral hemorrhagic fever may also occur.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row5\" class=\"row_heading level0 row5\" >5</th>\n",
              "      <td id=\"T_b4c57_row5_col0\" class=\"data row5 col0\" >Archaeal virus</td>\n",
              "      <td id=\"T_b4c57_row5_col1\" class=\"data row5 col1\" >An archaeal virus is a virus that infects and replicates in archaea, a domain of unicellular, prokaryotic organisms. Archaeal viruses, like their hosts, are found worldwide, including in extreme environments inhospitable to most life such as acidic hot springs, highly saline bodies of water, and at the bottom of the ocean. They have been also found in the human body. The first known archaeal virus was described in 1974 and since then, a large diversity of archaeal viruses have been discovered, many possessing unique characteristics not found in other viruses. Little is known about their biological processes, such as how they replicate, but they are believed to have many independent origins, some of which likely predate the last archaeal common ancestor (LACA).Much of the diversity observed in archaeal viruses is their morphology. Their complete bodies, called virions, come in many different forms, including being shaped like spindles or lemons, rods, bottles, droplets, and coils. Some contain a viral envelope, a lipid membrane that surrounds the viral capsid, which stores the viral genome. In some cases, the envelope surrounds the genome inside the capsid. All known archaeal viruses have genomes made of deoxyribonucleic acid (DNA), but some may contain ribonucleic acid (RNA) genomes. Almost all that have been identified contain double-stranded DNA genomes, a small minority having single-stranded DNA genomes. A large portion of the genes encoded by archaeal viruses have no known function or homology to any other genes.Compared to bacterial and eukaryotic viruses, few archaeal viruses have been described in detail. Despite this, those that have been studied are highly diverse and classified to more than 20 families, many of which show no relation to any other known viruses. In general, all archaeal viruses can be placed into two broad groups: those that are related to bacterial and eukaryotic viruses and those that are not. The former includes viruses found in the realms Duplodnaviria and Varidnaviria, which likely have ancient origins preceding the LACA, and the latter includes the realm Adnaviria and all archaeal virus families unassigned to higher taxa, which are thought to have more recent origins from non-viral mobile genetic elements such as plasmids.\n",
              "How archaeal viruses interact with their hosts and the environment is largely unknown. Many establish a persistent infection, during which progeny are continually produced at a low rate without killing the host archaeon. Some have evolved alongside their hosts, adapting to the environments in which archaea live. For example, bicaudaviruses grow two tails on opposite ends of their bodies after they leave their host cell, which may help them find a new host in sparsely populated environments. In oceans, archaeal viruses are believed to play a major role in recycling nutrients, especially at the bottom of the ocean where they are a major cause of death. For some archaeal viruses in hypersaline environments, the level of salinity can affect infectivity and virus behavior.\n",
              "Research areas in archaeal virology include gaining a better understanding of their diversity and learning about their means of replication. Some environments, such as acidic hot springs, are almost exclusively populated by archaea, so these environments are highly useful for studying how archaeal viruses interact with their hosts. Because a large portion of their genes have no known function, there is a large reserve of genetic material to be explored. In the early decades of archaeal virus research, Wolfram Zillig and his colleagues discovered numerous archaeal virus families. Since 2000, methods such as metagenomics have identified many novel archaeal viruses, and methods such as cryogenic electron microscopy and gene synteny have helped to better understand their evolutionary history.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row6\" class=\"row_heading level0 row6\" >6</th>\n",
              "      <td id=\"T_b4c57_row6_col0\" class=\"data row6 col0\" >Astrovirology</td>\n",
              "      <td id=\"T_b4c57_row6_col1\" class=\"data row6 col1\" >Astrovirology is an emerging subdiscipline of astrobiology which aims to understand what role viruses played in the origin and evolution of life on Earth as well as the potential for viruses beyond Earth.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row7\" class=\"row_heading level0 row7\" >7</th>\n",
              "      <td id=\"T_b4c57_row7_col0\" class=\"data row7 col0\" >Bacteriophage Mu</td>\n",
              "      <td id=\"T_b4c57_row7_col1\" class=\"data row7 col1\" >Bacteriophage Mu, also known as mu phage or mu bacteriophage, is a muvirus (the first of its kind to be identified) of the family Myoviridae which has been shown to cause genetic transposition. It is of particular importance as its discovery in Escherichia coli by Larry Taylor was among the first observations of insertion elements in a genome. This discovery opened up the world to an investigation of transposable elements and their effects on a wide variety of organisms. While Mu was specifically involved in several distinct areas of research (including E. coli, maize, and HIV), the wider implications of transposition and insertion transformed the entire field of genetics.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row8\" class=\"row_heading level0 row8\" >8</th>\n",
              "      <td id=\"T_b4c57_row8_col0\" class=\"data row8 col0\" >Baltimore classification</td>\n",
              "      <td id=\"T_b4c57_row8_col1\" class=\"data row8 col1\" >Baltimore classification is a system used to classify viruses based on their manner of messenger RNA (mRNA) synthesis. By organizing viruses based on their manner of mRNA production, it is possible to study viruses that behave similarly as a distinct group. Seven Baltimore groups are described that take into consideration whether the viral genome is made of deoxyribonucleic acid (DNA) or ribonucleic acid (RNA), whether the genome is single- or double-stranded, and whether the sense of a single-stranded RNA genome is positive or negative.\n",
              "Baltimore classification also closely corresponds to the manner of replicating the genome, so Baltimore classification is useful for grouping viruses together for both transcription and replication. Certain subjects pertaining to viruses are associated with multiple, specific Baltimore groups, such as specific forms of translation of mRNA and the host range of different types of viruses. Structural characteristics such as the shape of the viral capsid, which stores the viral genome, and the evolutionary history of viruses are not necessarily related to Baltimore groups.\n",
              "Baltimore classification was created in 1971 by virologist David Baltimore. Since then, it has become common among virologists to use Baltimore classification alongside standard virus taxonomy, which is based on evolutionary history. In 2018 and 2019, Baltimore classification was partially integrated into virus taxonomy based on evidence that certain groups were descended from common ancestors. Various realms, kingdoms, and phyla now correspond to specific Baltimore groups.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row9\" class=\"row_heading level0 row9\" >9</th>\n",
              "      <td id=\"T_b4c57_row9_col0\" class=\"data row9 col0\" >Botourmiaviridae</td>\n",
              "      <td id=\"T_b4c57_row9_col1\" class=\"data row9 col1\" >Botourmiaviridae is a family of positive-strand RNA viruses which infect plants and fungi. The family includes four genera: Ourmiavirus, Botoulivirus, Magoulivirus and Scleroulivirus. Members of genus Ourmiavirus infect plants and the other genera infect fungi. The member viruses have genomes which range from 2900 to 4800 nucleotides.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row10\" class=\"row_heading level0 row10\" >10</th>\n",
              "      <td id=\"T_b4c57_row10_col0\" class=\"data row10 col0\" >Broad-spectrum antiviral drug</td>\n",
              "      <td id=\"T_b4c57_row10_col1\" class=\"data row10 col1\" >Broad-spectrum antivirals (BSAs) are a class of molecules or compounds, which inhibit the replication of a broad range of viruses (i.e. viruses belonging to two or more viral families). BSAs could be divided into experimental and investigational agents, and approved drugs. BSAs work by inhibiting viral proteins (such as polymerases and proteases) or by targeting host cell factors and processes exploited by different viruses during infection. As of 2021, there are 150 known BSAs in varying stages of development, effective against 78 human viruses. BSAs are potential candidates for treatment of emerging and re-emerging viruses, such as ebola, marburg, and SARS-CoV-2. Many BSAs show antiviral activity against other viruses than originally investigated (such as remdesivir and interferon alpha). Efforts in drug repurposing for SARS-CoV-2 is currently underway. A database of BSAs and viruses they inhibit could be found here (https://drugvirus.info/).</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row11\" class=\"row_heading level0 row11\" >11</th>\n",
              "      <td id=\"T_b4c57_row11_col0\" class=\"data row11 col0\" >Chimera (virus)</td>\n",
              "      <td id=\"T_b4c57_row11_col1\" class=\"data row11 col1\" >A chimera or chimeric virus is a virus that contains genetic material derived from two or more distinct viruses. It is defined by the Center for Veterinary Biologics (part of the U.S. Department of Agriculture's Animal and Plant Health Inspection Service) as a \"new hybrid microorganism created by joining nucleic acid fragments from two or more different microorganisms in which each of at least two of the fragments contain essential genes necessary for replication.\" The term genetic chimera had already been defined to mean: an individual organism whose body contained cell populations from different zygotes or an organism that developed from portions of different embryos. Chimeric flaviviruses have been created in an attempt to make novel live attenuated vaccines.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row12\" class=\"row_heading level0 row12\" >12</th>\n",
              "      <td id=\"T_b4c57_row12_col0\" class=\"data row12 col0\" >Contagium vivum fluidum</td>\n",
              "      <td id=\"T_b4c57_row12_col1\" class=\"data row12 col1\" >Contagium vivum fluidum (Latin: \"contagious living fluid\") was a phrase first used to describe a virus, and underlined its ability to slip through the finest-mesh filters then available, giving it almost liquid properties. Martinus Beijerinck (1851-1931), a Dutch microbiologist and botanist, first used the term when studying the tobacco mosaic virus, becoming convinced that the virus had a liquid nature.In 1892, Dmitri Ivanovsky had discovered that the cause of tobacco mosaic disease could pass through Chamberland's porcelain filter. He noted these findings but felt that they could only have resulted from a cracked filter. It was left to Beijerinck, in 1898, to put forward the idea that the pathogen was small enough to pass through the filter routinely used to trap bacteria.Ivanovsky, irked that Beijerinck had not cited him, recreated Beijerinck's experimental set-up and demonstrated that particles of ink were small enough to pass through the filter, thus leaving the particulate or fluid nature of the pathogen unresolved. The invention of the electron microscope in 1931 led to the first virus images in 1935 by the American biochemist and virologist Wendell Meredith Stanley. The second half of the 1900s resulted in the discovery of more than 2,000 virus species infecting animals, plants and bacteria.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row13\" class=\"row_heading level0 row13\" >13</th>\n",
              "      <td id=\"T_b4c57_row13_col0\" class=\"data row13 col0\" >Riccardo Cortese</td>\n",
              "      <td id=\"T_b4c57_row13_col1\" class=\"data row13 col1\" >Riccardo Cortese (Siena, Italy, March 29, 1944 - Basel, Switzerland, April 27, 2017) was an Italian scientist, entrepreneur, and innovator in the field of gene expression, drug discovery and genetic vaccines. His work led to the development of novel therapeutic strategies for the prevention and cure of viral infections, including HIV, HCV, Ebola and RSV. He pioneered a novel platform technology based on simian adenoviral vectors for prophylactic and therapeutic vaccines, and authored more than 300 publications in peer reviewed journals in the field of gene expression, transcriptional control, molecular virology and immunology.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row14\" class=\"row_heading level0 row14\" >14</th>\n",
              "      <td id=\"T_b4c57_row14_col0\" class=\"data row14 col0\" >Cossaviricota</td>\n",
              "      <td id=\"T_b4c57_row14_col1\" class=\"data row14 col1\" >Cossaviricota is a phylum of viruses.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row15\" class=\"row_heading level0 row15\" >15</th>\n",
              "      <td id=\"T_b4c57_row15_col0\" class=\"data row15 col0\" >Cressdnaviricota</td>\n",
              "      <td id=\"T_b4c57_row15_col1\" class=\"data row15 col1\" >Cressdnaviricota is a phylum of viruses with small, circular single-stranded DNA genomes and encoding rolling circle replication-initiation proteins with the N-terminal HUH endonuclease and C-terminal superfamily 3 helicase domains.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row16\" class=\"row_heading level0 row16\" >16</th>\n",
              "      <td id=\"T_b4c57_row16_col0\" class=\"data row16 col0\" >Cross-species transmission</td>\n",
              "      <td id=\"T_b4c57_row16_col1\" class=\"data row16 col1\" >Cross-species transmission (CST), also called interspecies transmission, host jump, or spillover, is the transmission of an infectious pathogen, such as a virus, between hosts belonging to different species. Once introduced into an individual of a new host species, the pathogen may cause disease for the new host and/or acquire the ability to infect other individuals of the same species, allowing it to spread through the new host population. The phenomenon is most commonly studied in virology, but cross-species transmission may also occur with bacterial pathogens or other types of microorganisms.Steps involved in the transfer of pathogens to new hosts include contact between the pathogen and the host; the successful infection of an initial individual host, which may lead to amplification and an outbreak; and the adaptation of the pathogen, within either the original or new host, which may render it capable of spreading efficiently between individuals in populations of the new host. The concept is important in understanding and controlling emerging infectious diseases in humans, especially those caused by viruses. Most viral diseases of humans are zoonotic in origin, having been historically transmitted to human populations from various animal species; examples include SARS, Ebola, swine flu, rabies, and avian influenza.The exact mechanisms which facilitate cross-species transmission vary by pathogen, and even for common diseases are often poorly understood. It is believed that viruses with high mutation rates are able to rapidly adapt to new hosts and thereby overcome host-specific immunological defenses, allowing their continued transmission. A host shifting event occurs when a strain that was previously zoonotic begins to circulate exclusively among the new host species.Pathogen transfer is most likely to occur between species which are frequently in close contact with each other. It can also occur indirectly between species with less frequent contact if facilitated by an intermediary species; for example, a reservoir species may transfer the virus to a vector species, which in turn transfers the virus to humans. The degree of phylogenetic relatedness between host species also influences the likelihood that a pathogen is transmitted between them, likely because of the similarity of the hosts' immunological defenses; for example, most human zoonotic transmissions come from other species of mammals. Pathogens of more distantly related species, on the other hand, such as plant viruses, may not be capable of infecting humans at all. Other factors influencing transmission rates include geographic proximity and intraspecies behaviors.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row17\" class=\"row_heading level0 row17\" >17</th>\n",
              "      <td id=\"T_b4c57_row17_col0\" class=\"data row17 col0\" >Disease X</td>\n",
              "      <td id=\"T_b4c57_row17_col1\" class=\"data row17 col1\" >Disease X is a placeholder name that was adopted by the World Health Organization (WHO) in February 2018 on their shortlist of blueprint priority diseases to represent a hypothetical, unknown pathogen that could cause a future epidemic. The WHO adopted the placeholder name to ensure that their planning was sufficiently flexible to adapt to an unknown pathogen (e.g., broader vaccines and manufacturing facilities). Director of the US National Institute of Allergy and Infectious Diseases Anthony Fauci stated that the concept of Disease X would encourage WHO projects to focus their research efforts on entire classes of viruses (e.g., flaviviruses), instead of just individual strains (e.g., zika virus), thus improving WHO capability to respond to unforeseen strains. In 2020, it was speculated, including among some of the WHO's own expert advisors, that COVID-19, caused by SARS-CoV-2 virus strain, met the requirements to be the first Disease X.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row18\" class=\"row_heading level0 row18\" >18</th>\n",
              "      <td id=\"T_b4c57_row18_col0\" class=\"data row18 col0\" >Duplodnaviria</td>\n",
              "      <td id=\"T_b4c57_row18_col1\" class=\"data row18 col1\" >Duplodnaviria is a realm of viruses that includes all double-stranded DNA viruses that encode the HK97 fold major capsid protein. The HK97 fold major capsid protein (HK97 MCP) is the primary component of the viral capsid, which stores the viral deoxyribonucleic acid (DNA). Viruses in the realm also share a number of other characteristics, such as an icosahedral capsid, an opening in the viral capsid called a portal, a protease enzyme that empties the inside of the capsid prior to DNA packaging, and a terminase enzyme that packages viral DNA into the capsid.\n",
              "Duplodnaviria was established in 2019 based on the shared characteristics of viruses in the realm. There are two groups of viruses in Duplodnaviria: tailed bacteriophages of the order Caudovirales, which infect prokaryotes, and herpesviruses of the order Herpesvirales, which infect animals. Tailed bacteriophages are very diverse and ubiquitous worldwide, and they may be the oldest lineage of viruses. Herpesviruses either share a common ancestor with tailed bacteriophages or are a breakaway group from within Caudovirales.\n",
              "Tailed bacteriophages are important in marine ecology by recycling nutrients in organic material from their hosts and are the focus of much research, and herpesviruses are associated with a variety of diseases in animals, including humans. A common feature among viruses in Duplodnaviria is that many are able to persist in their host for long periods of time without replicating while still being able to resurface in the future. Examples of this include the herpes simplex virus, which causes recurring infections, and the varicella zoster virus, which initially causes chickenpox early in life then shingles later in life.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row19\" class=\"row_heading level0 row19\" >19</th>\n",
              "      <td id=\"T_b4c57_row19_col0\" class=\"data row19 col0\" >Duplornaviricota</td>\n",
              "      <td id=\"T_b4c57_row19_col1\" class=\"data row19 col1\" >Duplornaviricota is a phylum of RNA viruses, which contains all double-stranded RNA viruses, except for those in phylum Pisuviricota. Characteristic of the group is a viral capsid composed of 60 homo- or heterodimers of capsid protein on a pseudo-T=2 lattice. Duplornaviruses infect both prokaryotes and eukaryotes. The name of the group derives from Italian duplo which means double (a reference to double-stranded), rna for the type of virus, and -viricota which is the suffix for a virus phylum.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row20\" class=\"row_heading level0 row20\" >20</th>\n",
              "      <td id=\"T_b4c57_row20_col0\" class=\"data row20 col0\" >Durnavirales</td>\n",
              "      <td id=\"T_b4c57_row20_col1\" class=\"data row20 col1\" >Durnavirales is an order of double-stranded RNA viruses which infect eukaryotes. The name of the group derives from Italian duplo which means double (a reference to double-stranded), rna for the type of virus, and -virales which is the suffix for a virus order.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row21\" class=\"row_heading level0 row21\" >21</th>\n",
              "      <td id=\"T_b4c57_row21_col0\" class=\"data row21 col0\" >Emergent virus</td>\n",
              "      <td id=\"T_b4c57_row21_col1\" class=\"data row21 col1\" >An emergent virus (or emerging virus) is a virus that is either newly appeared, notably increasing in incidence/geographic range or has the potential to increase in the near future. Emergent viruses are a leading cause of emerging infectious diseases and raise public health challenges globally, given their potential to cause outbreaks of disease which can lead to epidemics and pandemics. As well as causing disease, emergent viruses can also have severe economic implications. Recent examples include the SARS-related coronaviruses, which have caused the 2002-2004 outbreak of SARS (SARS-CoV-1) and the 2019–21 pandemic of COVID-19 (SARS-CoV-2). Other examples include the human immunodeficiency virus which causes HIV/AIDS; the viruses responsible for Ebola; the H5N1 influenza virus responsible for avian flu; and H1N1/09, which caused the 2009 swine flu pandemic (an earlier emergent strain of H1N1 caused the 1918 Spanish flu pandemic). Viral emergence in humans is often a consequence of zoonosis, which involves a cross-species jump of a viral disease into humans from other animals. As zoonotic viruses exist in animal reservoirs, they are much more difficult to eradicate and can therefore establish persistent infections in human populations.Emergent viruses should not be confused with re-emerging viruses or newly detected viruses. A re-emerging virus is generally considered to be a previously appeared virus that is experiencing a resurgence, for example measles. A newly detected virus is a previously unrecognized virus that had been circulating in the species as endemic or epidemic infections. Newly detected viruses may have escaped classification because they left no distinctive clues, and/or could not be isolated or propagated in cell culture. Examples include human rhinovirus (a leading cause of common colds which was first identified in 1956), hepatitis C (eventually identified in 1989),  and human metapneumovirus (first described in 2001, but thought to have been circulating since the 19th century). As the detection of such viruses is technology driven, the number reported is likely to expand.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row22\" class=\"row_heading level0 row22\" >22</th>\n",
              "      <td id=\"T_b4c57_row22_col0\" class=\"data row22 col0\" >Epidemiology of Hepatitis D</td>\n",
              "      <td id=\"T_b4c57_row22_col1\" class=\"data row22 col1\" >The epidemiology of hepatitis D occurs worldwide. Although the figures are disputed, a recent systematic review suggests that up to 60 million individuals could be infected. The major victims are the carriers of the hepatitis B surface antigen (HBsAg), who become superinfected by the HDV, and intravenous drug users who are the group at highest risk. The infection usually results in liver damage (hepatitis D); this is most often a chronic and severe hepatitis rapidly conducive to cirrhosis.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row23\" class=\"row_heading level0 row23\" >23</th>\n",
              "      <td id=\"T_b4c57_row23_col0\" class=\"data row23 col0\" >Genetically modified vaccine</td>\n",
              "      <td id=\"T_b4c57_row23_col1\" class=\"data row23 col1\" >Most vaccines consist of viruses that have been attenuated, disabled, weakened or killed in some way so that their virulent properties are no longer effective. A simple\n",
              "genetically modified vaccine, based on a thymidine kinase deficient mutant of pseudorabies virus was reportedly available as early as 2001 as a commercial vaccine to control Aujeszky's disease in Europe, North America and Japan.\n",
              "\n",
              "\n",
              "== References ==</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row24\" class=\"row_heading level0 row24\" >24</th>\n",
              "      <td id=\"T_b4c57_row24_col0\" class=\"data row24 col0\" >Genetically modified virus</td>\n",
              "      <td id=\"T_b4c57_row24_col1\" class=\"data row24 col1\" >A genetically modified virus is a virus that has been altered or generated using biotechnology methods, and remains capable of infection. Genetic modification involves the directed insertion, deletion, artificial synthesis or change of nucleotide bases in viral genomes. Genetically modified viruses are mostly generated by the insertion of foreign genes intro viral genomes for the purposes of biomedical, agricultural, bio-control, or technological objectives. The terms genetically modified virus and genetically engineered virus are used synonymously.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row25\" class=\"row_heading level0 row25\" >25</th>\n",
              "      <td id=\"T_b4c57_row25_col0\" class=\"data row25 col0\" >Ghabrivirales</td>\n",
              "      <td id=\"T_b4c57_row25_col1\" class=\"data row25 col1\" >Ghabrivirales is an order of double-stranded RNA viruses. It is the only order in the class Chrysmotiviricetes. The name of the class is a portmanteau of member families: chrysoviridae, megabirnaviridae, and totiviridae; and -viricetes which is the suffix for a virus class. The name of the order derives from Said Ghabrial, a pioneering researcher who studied viruses in this order, and -virales which is the suffix for a virus order.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row26\" class=\"row_heading level0 row26\" >26</th>\n",
              "      <td id=\"T_b4c57_row26_col0\" class=\"data row26 col0\" >Hepelivirales</td>\n",
              "      <td id=\"T_b4c57_row26_col1\" class=\"data row26 col1\" >Hepelivirales is an order of viruses.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row27\" class=\"row_heading level0 row27\" >27</th>\n",
              "      <td id=\"T_b4c57_row27_col0\" class=\"data row27 col0\" >Horizontal Environmental Genetic Alteration Agents</td>\n",
              "      <td id=\"T_b4c57_row27_col1\" class=\"data row27 col1\" >Horizontal Environmental Genetic Alteration Agents (HEGAAs) are any artificially developed agents that are engineered to edit the genome of eukaryotic species they infect when intentionally dispersed into the environment (outside of contained facilities such as laboratories or hospitals).</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row28\" class=\"row_heading level0 row28\" >28</th>\n",
              "      <td id=\"T_b4c57_row28_col0\" class=\"data row28 col0\" >Human virome</td>\n",
              "      <td id=\"T_b4c57_row28_col1\" class=\"data row28 col1\" >The human virome is the total collection of viruses in and on the human body. Viruses in the human body may infect both human cells and other microbes such as bacteria (as with bacteriophages). Some viruses cause disease, while others may be asymptomatic. Certain viruses are also integrated into the human genome as proviruses or endogenous viral elements.Viruses evolve rapidly and hence the human virome changes constantly. Every human being has a unique virome with a unique balance of species. Lifestyle, age, geographic location, and even the season of the year can affect an individual's exposure to viruses, and one's susceptibility to any disease that might be caused by those viruses is also affected by pre-existing immunity and both viral and human genetics.The human virome is far from being completely explored and new viruses are discovered frequently. Unlike the roughly 40 trillion bacteria in a typical human microbiome, an estimate of the number of viral particles in a healthy adult human is not yet available, although virions generally outnumber individual bacteria 10:1 in nature. Studying the virome is thought to provide an understanding of microbes in general and how they affect human health and disease.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row29\" class=\"row_heading level0 row29\" >29</th>\n",
              "      <td id=\"T_b4c57_row29_col0\" class=\"data row29 col0\" >Human viruses in water</td>\n",
              "      <td id=\"T_b4c57_row29_col1\" class=\"data row29 col1\" >Viruses are a major cause of human waterborne and water-related diseases. Waterborne diseases are caused by water that is contaminated by human and animal urine and feces that contain pathogenic microorganisms. A subject can get infected through contact with or consumption of the contaminated water. Viruses affect all living organisms from single cellular plants, bacteria and animal to the highest forms of plants and animals including human beings. Within a specific kingdom ( Plantae, Animalia, Fungi etc) the localization of viruses colonizing the host can vary: Some human viruses, for example, HIV, colonizes only the immune system, while influenza viruses on the other hand can colonize either the upper respiratory tract or the lower respiratory tract depending on the type (human Influenza virus or avian influenza viruses respectively).  Different viruses can have different routes of transmission; for example, HIV is directly transferred by contaminated body fluids from an infected host into the tissue or bloodstream of a new host while influenza is airborne and transmitted through inhalation of contaminated air containing viral particles by a new host. Research has also suggested that solid surface plays a role in the transmission of water viruses. In a experiments that used E.coli phages, Qβ, fr, T4, and MS2 confirmed that viruses survive on a solid surface longer compared to when they are in water. Because of this adaptation to survive longer on solid surfaces, viruses now have a prolonged opportunities to infect humans.  Enteric viruses primarily infect the intestinal tract through ingestion of food and water contaminated with viruses of fecal origin. Some viruses can be transmitted through all three routes of transmission.\n",
              "Water virology started about half a century ago when scientists attempted to detect the polio virus in water samples.  Since then, other pathogenic viruses that are responsible for gastroenteritis, hepatitis, and many other virus strains have replaced enteroviruses as the main aim for detection in the water environment.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row30\" class=\"row_heading level0 row30\" >30</th>\n",
              "      <td id=\"T_b4c57_row30_col0\" class=\"data row30 col0\" >Kitrinoviricota</td>\n",
              "      <td id=\"T_b4c57_row30_col1\" class=\"data row30 col1\" >Kitrinoviricota is a phylum of RNA viruses that includes all positive-strand RNA viruses that infect eukaryotes and are not members of the phylum Pisuviricota or Lenarviricota. The name of the group derives from Greek κίτρινος (kítrinos), which means yellow (a reference to yellow fever virus), and -viricota, which is the suffix for a virus phylum.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row31\" class=\"row_heading level0 row31\" >31</th>\n",
              "      <td id=\"T_b4c57_row31_col0\" class=\"data row31 col0\" >Lefavirales</td>\n",
              "      <td id=\"T_b4c57_row31_col1\" class=\"data row31 col1\" >Lefavirales is an order of viruses.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row32\" class=\"row_heading level0 row32\" >32</th>\n",
              "      <td id=\"T_b4c57_row32_col0\" class=\"data row32 col0\" >Lenarviricota</td>\n",
              "      <td id=\"T_b4c57_row32_col1\" class=\"data row32 col1\" >Lenarviricota is a phylum of RNA viruses that includes all positive-strand RNA viruses that infect prokaryotes. Some members also infect eukaryotes. Most of these viruses do not have capsids, except for the genus Ourmiavirus. The name of the group is a syllabic abbreviation of the names of founding member families \"Leviviridae and Narnaviridae\" with the suffix -viricota, denoting a virus phylum.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row33\" class=\"row_heading level0 row33\" >33</th>\n",
              "      <td id=\"T_b4c57_row33_col0\" class=\"data row33 col0\" >Marine viruses</td>\n",
              "      <td id=\"T_b4c57_row33_col1\" class=\"data row33 col1\" >Marine viruses are defined by their habitat as viruses that are found in marine environments, that is, in the saltwater of seas or oceans or the brackish water of coastal estuaries. Viruses are small infectious agents that can only replicate inside the living cells of a host organism, because they need the replication machinery of the host to do so. They can infect all types of life forms, from animals and plants to microorganisms, including bacteria and archaea.When not inside a cell or in the process of infecting a cell, viruses exist in the form of independent particles called virions. A virion contains a genome (long molecules that carry genetic information in the form of either DNA or RNA) surrounded by a capsid (a protein coat protecting the genetic material). The shapes of these virus particles range from simple helical and icosahedral forms for some virus species to more complex structures for others. Most virus species have virions that are too small to be seen with an optical microscope. The average virion is about one one-hundredth the linear size of the average bacterium.\n",
              "A teaspoon of seawater typically contains about fifty million viruses.  Most of these viruses are bacteriophages which infect and destroy marine bacteria and control the growth of phytoplankton at the base of the marine food web. Bacteriophages are harmless to plants and animals but are essential to the regulation of marine ecosystems. They supply key mechanisms for recycling ocean carbon and nutrients. In a process known as the viral shunt, organic molecules released from dead bacterial cells stimulate fresh bacterial and algal growth. In particular, the breaking down of bacteria by viruses (lysis) has been shown to enhance nitrogen cycling and stimulate phytoplankton growth. Viral activity also affects the biological pump, the process which sequesters carbon in the deep ocean. By increasing the amount of respiration in the oceans, viruses are indirectly responsible for reducing the amount of carbon dioxide in the atmosphere by approximately 3 gigatonnes of carbon per year.\n",
              "Marine microorganisms make up about 70% of the total marine biomass. It is estimated marine viruses kill 20% of this biomass every day. Viruses are the main agents responsible for the rapid destruction of harmful algal blooms which often kill other marine life. The number of viruses in the oceans decreases further offshore and deeper into the water, where there are fewer host organisms. Viruses are an important natural means of transferring genes between different species, which increases genetic diversity and drives evolution. It is thought viruses played a central role in early evolution before the diversification of bacteria, archaea and eukaryotes, at the time of the last universal common ancestor of life on Earth. Viruses are still one of the largest areas of unexplored genetic diversity on Earth.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row34\" class=\"row_heading level0 row34\" >34</th>\n",
              "      <td id=\"T_b4c57_row34_col0\" class=\"data row34 col0\" >Martellivirales</td>\n",
              "      <td id=\"T_b4c57_row34_col1\" class=\"data row34 col1\" >Martellivirales is an order of viruses.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row35\" class=\"row_heading level0 row35\" >35</th>\n",
              "      <td id=\"T_b4c57_row35_col0\" class=\"data row35 col0\" >Mosaic virus</td>\n",
              "      <td id=\"T_b4c57_row35_col1\" class=\"data row35 col1\" >A mosaic virus is any virus that causes infected plant foliage to have a mottled appearance. Such viruses come from a variety of unrelated lineages and consequently there is no taxon that unites all mosaic viruses. All the symptoms of each virus or complex appear the same, so the only way to know exactly which viral subtype is infecting a plant is to send it to a local extension service. Alternatively, commercial testing kits exist for settings like greenhouses.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row36\" class=\"row_heading level0 row36\" >36</th>\n",
              "      <td id=\"T_b4c57_row36_col0\" class=\"data row36 col0\" >Naldaviricetes</td>\n",
              "      <td id=\"T_b4c57_row36_col1\" class=\"data row36 col1\" >Naldaviricetes is a class of viruses, which infect arthropods. Members of Naldaviricetes are characterized by large enveloped rod-shaped virions, circular double-stranded DNA genomes, and replication in the nucleus of the host cell. All of them (including the unassigned Polydnaviridae family) share a set of unique genes not found in other viruses, which include the presence of multiple interspersed direct repeats, various subunits of DNA polymerase and RNA polymerase, four late expression factor genes, and infectivity factor genes suggesting a common host entry mechanism.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row37\" class=\"row_heading level0 row37\" >37</th>\n",
              "      <td id=\"T_b4c57_row37_col0\" class=\"data row37 col0\" >Neurotropic virus</td>\n",
              "      <td id=\"T_b4c57_row37_col1\" class=\"data row37 col1\" >A neurotropic virus is a virus that is capable of infecting nerve tissue.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row38\" class=\"row_heading level0 row38\" >38</th>\n",
              "      <td id=\"T_b4c57_row38_col0\" class=\"data row38 col0\" >Nodamuvirales</td>\n",
              "      <td id=\"T_b4c57_row38_col1\" class=\"data row38 col1\" >Nodamuvirales is an order of positive-strand RNA viruses which infect eukaryotes.  The name of the group is a contraction of  \"Nodamura virus\" and -virales which is the suffix for a virus order.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row39\" class=\"row_heading level0 row39\" >39</th>\n",
              "      <td id=\"T_b4c57_row39_col0\" class=\"data row39 col0\" >Non-cellular life</td>\n",
              "      <td id=\"T_b4c57_row39_col1\" class=\"data row39 col1\" >Non-cellular life, or acellular life is life that exists without a cellular structure for at least part of its life cycle. Historically, most (descriptive) definitions of life postulated that an organism must be composed of one or more cells, but this is no longer considered necessary, and modern criteria allow for forms of life based on other structural arrangements.The primary candidates for non-cellular life are viruses. Some biologists consider viruses to be organisms, but others do not. Their primary objection is that no known viruses are capable of autonomous reproduction: they must rely on cells to copy them.Engineers sometimes use the term \"artificial life\" to refer to software and robots inspired by biological processes, but these do not satisfy any biological definition of life.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row40\" class=\"row_heading level0 row40\" >40</th>\n",
              "      <td id=\"T_b4c57_row40_col0\" class=\"data row40 col0\" >Novel virus</td>\n",
              "      <td id=\"T_b4c57_row40_col1\" class=\"data row40 col1\" >A novel virus is a virus that has not previously been recorded. It can be a virus that is isolated from its natural reservoir or isolated as the result of spread to an animal or human host where the virus had not been identified before. It can be an emergent virus, one that represents a new virus, but it can also be an extant virus that has not been previously identified.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row41\" class=\"row_heading level0 row41\" >41</th>\n",
              "      <td id=\"T_b4c57_row41_col0\" class=\"data row41 col0\" >Nucleocytoviricota</td>\n",
              "      <td id=\"T_b4c57_row41_col1\" class=\"data row41 col1\" >Nucleocytoviricota is a phylum of viruses. Members of the phylum are also known as the nucleocytoplasmic large DNA viruses (NCLDV), which serves as the basis of the name of the phylum with the suffix -viricota for virus phylum. These viruses are referred to as nucleocytoplasmic because they are often able to replicate in both the host's cell nucleus and cytoplasm.The phylum is notable for containing the giant viruses. There are nine families of NCLDVs that all share certain genomic and structural characteristics; however, it is uncertain whether the similarities of the different families of this group have a common viral ancestor. One feature of this group is a large genome and the presence of many genes involved in DNA repair, DNA replication, transcription, and translation. Typically, viruses with smaller genomes do not contain genes for these processes. Most of the viruses in this family also replicate in both the host's nucleus and cytoplasm, thus the name nucleocytoplasmic.\n",
              "There are 47 NCLDV core genes currently recognised. These include four key proteins involved in DNA replication and repair: the enzymes DNA polymerase family B, the topoisomerase II A, the FLAP endonuclease and the processing factor proliferating cell nuclear antigen. Other proteins include DNA dependent RNA polymerase II and transcription factor II B.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row42\" class=\"row_heading level0 row42\" >42</th>\n",
              "      <td id=\"T_b4c57_row42_col0\" class=\"data row42 col0\" >Oncovirus</td>\n",
              "      <td id=\"T_b4c57_row42_col1\" class=\"data row42 col1\" >An oncovirus or oncogenic virus is a virus that can cause cancer. This term originated from studies of acutely transforming retroviruses in the 1950–60s, when the term \"oncornaviruses\" was used to denote their RNA virus origin. With the letters \"RNA\" removed, it now refers to any virus with a DNA or RNA genome causing cancer and is synonymous with \"tumor virus\" or \"cancer virus\". The vast majority of human and animal viruses do not cause cancer, probably because of longstanding co-evolution between the virus and its host. Oncoviruses have been important not only in epidemiology, but also in investigations of cell cycle control mechanisms such as the retinoblastoma protein.\n",
              "The World Health Organization's International Agency for Research on Cancer estimated that in 2002, infection caused 17.8% of human cancers, with 11.9% caused by one of seven viruses. A 2020 study of 2,658 samples from 38 different types of cancer found that 16% were associated with a virus. These cancers might be easily prevented through vaccination (e.g., papillomavirus vaccines), diagnosed with simple blood tests, and treated with less-toxic antiviral compounds.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row43\" class=\"row_heading level0 row43\" >43</th>\n",
              "      <td id=\"T_b4c57_row43_col0\" class=\"data row43 col0\" >Orphan virus</td>\n",
              "      <td id=\"T_b4c57_row43_col1\" class=\"data row43 col1\" >An orphan virus is a virus that is not associated with a disease but may possess pathogenicity.Some orphan viruses include adeno-associated virus (Parvoviridae), human herpesvirus 7 (Herpesviridae), human foamy virus (Retroviridae), Human Reovirus (Reoviridae), hepatitis G (Flaviviridae), and TT virus (Anelloviridae). \n",
              "Gilbert Dalldorf, a pathologist who showed that Coxsackie viruses paralyze mice but not humans, indicated that the term ‘orphan’ was created \"in a moment of conviviality\" by a group of virologists.Many enteroviruses are referred to as ECHO, enteric cytopathic human orphan viruses, because they were originally not associated with any disease. Even though many of them are associated with severe diseases, the name ECHO still continues to be used.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row44\" class=\"row_heading level0 row44\" >44</th>\n",
              "      <td id=\"T_b4c57_row44_col0\" class=\"data row44 col0\" >Orthornavirae</td>\n",
              "      <td id=\"T_b4c57_row44_col1\" class=\"data row44 col1\" >Orthornavirae is a kingdom of viruses that have genomes made of ribonucleic acid (RNA), those genomes encoding an RNA-dependent RNA polymerase (RdRp). The RdRp is used to transcribe the viral RNA genome into messenger RNA (mRNA) and to replicate the genome. Viruses in this kingdom also share a number of characteristics involving evolution, including high rates of genetic mutations, recombinations, and reassortments.\n",
              "Viruses in Orthornavirae belong to the realm Riboviria. They are descended from a common ancestor that may have been a non-viral molecule that encoded a reverse transcriptase instead of an RdRp for replication. The kingdom is subdivided into five phyla that separate member viruses based on their genome type, host range, and genetic similarity. Viruses with three genome types are included: positive-strand RNA viruses, negative-strand RNA viruses, and double-stranded RNA viruses. \n",
              "Many of the most widely known viral diseases are caused by RNA viruses in the kingdom, which includes coronaviruses, the Ebola virus, influenza viruses, the measles virus, and the rabies virus. The first virus to be discovered, tobacco mosaic virus, belongs to the kingdom. In modern history, RdRp-encoding RNA viruses have caused numerous disease outbreaks, and they infect many economically important crops. Most eukaryotic viruses, including most human, animal, and plant viruses, are RdRp-encoding RNA viruses. In contrast, there are relatively few prokaryotic viruses in the kingdom.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row45\" class=\"row_heading level0 row45\" >45</th>\n",
              "      <td id=\"T_b4c57_row45_col0\" class=\"data row45 col0\" >Papovaviricetes</td>\n",
              "      <td id=\"T_b4c57_row45_col1\" class=\"data row45 col1\" >Papovaviricetes is a class of viruses. The class shares the name of an abolished family, Papovaviridae, which was split in 1999 into the two families Papillomaviridae and Polyomaviridae. The class was established in 2019 and takes its name from the former family.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row46\" class=\"row_heading level0 row46\" >46</th>\n",
              "      <td id=\"T_b4c57_row46_col0\" class=\"data row46 col0\" >Pimascovirales</td>\n",
              "      <td id=\"T_b4c57_row46_col1\" class=\"data row46 col1\" >Pimascovirales is an order of viruses. The term is a portmanteau of a portmanteau of pitho-, irido-, marseille-, and ascoviruses.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row47\" class=\"row_heading level0 row47\" >47</th>\n",
              "      <td id=\"T_b4c57_row47_col0\" class=\"data row47 col0\" >Pisoniviricetes</td>\n",
              "      <td id=\"T_b4c57_row47_col1\" class=\"data row47 col1\" >Pisoniviricetes is a class of positive-strand RNA viruses which infect eukaryotes.  A characteristic of the group is a conserved 3C-like protease from the PA clan of proteases for processing the translated polyprotein. The name of the group is a portmanteau of member orders \"picornavirales, sobelivirales, nidovirales\" and -viricetes which is the suffix for a virus class.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row48\" class=\"row_heading level0 row48\" >48</th>\n",
              "      <td id=\"T_b4c57_row48_col0\" class=\"data row48 col0\" >Pisuviricota</td>\n",
              "      <td id=\"T_b4c57_row48_col1\" class=\"data row48 col1\" >Pisuviricota is a phylum of RNA viruses that includes all positive-strand and double-stranded RNA viruses that infect eukaryotes and are not members of the phylum Kitrinoviricota, Lenarviricota or Duplornaviricota. The name of the group is a syllabic abbreviation of “picornavirus supergroup” with the suffix -viricota, indicating a virus phylum. Phylogenetic analyses suggest that Birnaviridae and Permutotetraviridae, both currently unassigned to a phylum in Orthornavirae, also belong to this phylum and that both are sister groups. Another proposed family of the phylum is unassigned Polymycoviridae in Riboviria.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row49\" class=\"row_heading level0 row49\" >49</th>\n",
              "      <td id=\"T_b4c57_row49_col0\" class=\"data row49 col0\" >Pleolipoviridae</td>\n",
              "      <td id=\"T_b4c57_row49_col1\" class=\"data row49 col1\" >Pleolipoviridae is a family of DNA viruses that infect archaea.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row50\" class=\"row_heading level0 row50\" >50</th>\n",
              "      <td id=\"T_b4c57_row50_col0\" class=\"data row50 col0\" >Pokkesviricetes</td>\n",
              "      <td id=\"T_b4c57_row50_col1\" class=\"data row50 col1\" >Pokkesviricetes is a class of viruses.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row51\" class=\"row_heading level0 row51\" >51</th>\n",
              "      <td id=\"T_b4c57_row51_col0\" class=\"data row51 col0\" >Preplasmiviricota</td>\n",
              "      <td id=\"T_b4c57_row51_col1\" class=\"data row51 col1\" >Preplasmiviricota is a phylum of viruses.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row52\" class=\"row_heading level0 row52\" >52</th>\n",
              "      <td id=\"T_b4c57_row52_col0\" class=\"data row52 col0\" >Reporter virus particles</td>\n",
              "      <td id=\"T_b4c57_row52_col1\" class=\"data row52 col1\" >Reporter virus particles (RVPs) are replication-incompetent virus particles engineered to express one or more reporter genes upon infecting susceptible cells. Since the RVP genome lacks genes essential for viral replication, RVPs are capable of only a single round of infection. Thus they are safe to work with under BSL-2 conditions, enabling the study of highly pathogenic viruses using standard laboratory facilities. Expression of a reporter such as luciferase can provide a quantitative readout of infection. With proper design and quality control, RVPs remain stable under common assay conditions and yield reproducible results that correlate with those obtained from live virus. These qualities make RVPs a safer and faster alternative to plaque assays, and especially well-suited for high-throughput applications. RVPs offer flexibility for different uses, as they are antigenically identical to wild-type virus, and can be engineered with various proteins or express mutant envelopes to study infectivity or antigenicity.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row53\" class=\"row_heading level0 row53\" >53</th>\n",
              "      <td id=\"T_b4c57_row53_col0\" class=\"data row53 col0\" >Revtraviricetes</td>\n",
              "      <td id=\"T_b4c57_row53_col1\" class=\"data row53 col1\" >Revtraviricetes is a class of viruses that contains all viruses that encode a reverse transcriptase. The group includes all ssRNA-RT viruses (including the retroviruses) and dsDNA-RT viruses. It is the sole class in the phylum Artverviricota, which is the sole phylum in the kingdom Pararnavirae. The name of the group is a portmanteau of \"reverse transcriptase\" and -viricetes which is the suffix for a virus class.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row54\" class=\"row_heading level0 row54\" >54</th>\n",
              "      <td id=\"T_b4c57_row54_col0\" class=\"data row54 col0\" >Ribozyviria</td>\n",
              "      <td id=\"T_b4c57_row54_col1\" class=\"data row54 col1\" >Ribozyviria is a realm of satellite nucleic acids. Established in ICTV TaxoProp 2020.012D, the realm is named after the presence of genomic and antigenomic ribozymes of the Deltavirus type. Additional common features include a rod-like structure, a RNA-binding \"delta antigen\" encoded in the genome, and animal hosts. Furthermore, the size range of the genomes of these viruses is between around 1547-1735nt, they either encode a hammerhead ribozyme or a hepatitis delta virus ribozyme, and their coding capacity only involves one conserved protein. Most lineages of this realm are poorly understood, the notable exception being members of the genus Deltavirus, the causal agents of Hepatitis D in humans. This realm of viruses appears to have derived from retrozymes (a family of retrotransposons) or one of the other groups of mobile genetic elements that descended from retrozymes (i.e. viroids and satellites). But it was also proposed that they may have originated from viroids with capsid protein capture.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row55\" class=\"row_heading level0 row55\" >55</th>\n",
              "      <td id=\"T_b4c57_row55_col0\" class=\"data row55 col0\" >Robovirus</td>\n",
              "      <td id=\"T_b4c57_row55_col1\" class=\"data row55 col1\" >A robovirus is a zoonotic virus that is transmitted by a rodent vector (i.e., rodent borne).Roboviruses mainly belong to the virus families Arenaviridae and Hantaviridae.   Like arbovirus (arthropod borne) and tibovirus (tick borne) the name refers to its method of transmission, known as its vector. This is distinguished from a clade, which groups around a common ancestor. Some scientists now refer to arbovirus and robovirus together with the term ArboRobo-virus.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row56\" class=\"row_heading level0 row56\" >56</th>\n",
              "      <td id=\"T_b4c57_row56_col0\" class=\"data row56 col0\" >Ruhugu virus</td>\n",
              "      <td id=\"T_b4c57_row56_col1\" class=\"data row56 col1\" >Ruhugu virus, scientific name Rubivirus ruteetense, is a species of virus in the genus Rubivirus. It was discovered in 2019 in healthy Ugandan bats. It belongs to the family of Matonaviridae, a single-stranded RNA of positive polarity which is enclosed by an icosahedral capsid.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row57\" class=\"row_heading level0 row57\" >57</th>\n",
              "      <td id=\"T_b4c57_row57_col0\" class=\"data row57 col0\" >Rustrela virus</td>\n",
              "      <td id=\"T_b4c57_row57_col1\" class=\"data row57 col1\" >Rustrela virus, scientific name Rubivirus strelense, is a species of virus in the genus Rubivirus.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row58\" class=\"row_heading level0 row58\" >58</th>\n",
              "      <td id=\"T_b4c57_row58_col0\" class=\"data row58 col0\" >Shotokuvirae</td>\n",
              "      <td id=\"T_b4c57_row58_col1\" class=\"data row58 col1\" >Shotokuvirae is a kingdom of viruses.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row59\" class=\"row_heading level0 row59\" >59</th>\n",
              "      <td id=\"T_b4c57_row59_col0\" class=\"data row59 col0\" >Sobelivirales</td>\n",
              "      <td id=\"T_b4c57_row59_col1\" class=\"data row59 col1\" >Sobelivirales is an order of RNA viruses which infect eukaryotes. Member viruses have a positive-sense single-stranded RNA genome. The name of the group is a portmanteau of member orders \"sobemovirus-like\" and -virales which is the suffix for a virus order.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row60\" class=\"row_heading level0 row60\" >60</th>\n",
              "      <td id=\"T_b4c57_row60_col0\" class=\"data row60 col0\" >Spillover infection</td>\n",
              "      <td id=\"T_b4c57_row60_col1\" class=\"data row60 col1\" >Spillover infection, also known as pathogen spillover and spillover event, occurs when a reservoir population with a high pathogen prevalence comes into contact with a novel host population. The pathogen is transmitted from the reservoir population and may or may not be transmitted within the host population.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row61\" class=\"row_heading level0 row61\" >61</th>\n",
              "      <td id=\"T_b4c57_row61_col0\" class=\"data row61 col0\" >Stelpaviricetes</td>\n",
              "      <td id=\"T_b4c57_row61_col1\" class=\"data row61 col1\" >Stelpaviricetes is a class of non-enveloped, positive-strand RNA viruses which infect plants and animals. Characteristic of the group is a VPg protein attached to the 5'-end of the genome and a conserved 3C-like protease from the PA clan of proteases for processing the translated polyprotein. The name of the group is a syllabic abbreviation of member orders \"stellavirales, patatavirales\" with the suffix -viricetes denoting a virus class.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row62\" class=\"row_heading level0 row62\" >62</th>\n",
              "      <td id=\"T_b4c57_row62_col0\" class=\"data row62 col0\" >Tectiliviricetes</td>\n",
              "      <td id=\"T_b4c57_row62_col1\" class=\"data row62 col1\" >Tectiliviricetes (from tectivirid-like) is a class of viruses.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row63\" class=\"row_heading level0 row63\" >63</th>\n",
              "      <td id=\"T_b4c57_row63_col0\" class=\"data row63 col0\" >Tibovirus</td>\n",
              "      <td id=\"T_b4c57_row63_col1\" class=\"data row63 col1\" >Tibovirus Is term is often used to describe viruses that are transmitted by tick vectors. The word tibovirus is an acronym (TIck-BOrne virus). This falls within the superorder arthropod thus tibovirus is classified under Arthropod Borne virus (Arborvirus). For a person to acquire infection the tick must bite and feed for a sufficient period of time. The tiboviruses that affect humans are limited to within 3 families: Flaviviridae, Reoviridae, and Bunyaviridae.Flaviviridae:\n",
              "\n",
              "Virus: Tick-borne meningoencephalitis virus\n",
              "Disease: Tick-borne meningoencephalitis\n",
              "Vector: deer tick (Ixodes scapularis), Ixodes ricinus (Europe), Ixodes persulcatus (Russia + Asia))\n",
              "Region: Europe and Northern AsiaReoviridae:\n",
              "\n",
              "Virus: Colorado tick fever virus\n",
              "Disease: Colorado tick fever\n",
              "Vector: Dermacentor andersoni\n",
              "Region: US (West)Bunyaviridae:\n",
              "\n",
              "Virus: Crimean-Congo hemorrhagic fever virus\n",
              "Disease: Crimean-Congo hemorrhagic fever\n",
              "Vector: Hyalomma marginatum, Rhipicephalus bursa\n",
              "Region: Southern part of Asia, Northern Africa, Southern Europe\n",
              "Virus: Heartland virus\n",
              "Disease: Severe Febrile Illness\n",
              "Vector: Lone Star Tick (Amblyomma americanum)\n",
              "Region: Missouri and Tennessee, United States\n",
              "\n",
              "\n",
              "== References ==</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row64\" class=\"row_heading level0 row64\" >64</th>\n",
              "      <td id=\"T_b4c57_row64_col0\" class=\"data row64 col0\" >Tolivirales</td>\n",
              "      <td id=\"T_b4c57_row64_col1\" class=\"data row64 col1\" >Tolivirales is an order of RNA viruses which infect insects and plants. Member viruses have a positive-sense single-stranded RNA genome. The virions are non-enveloped, spherical, and have an icosahedral capsid. The name of the group is a syllabic abbreviation of \"tombusvirus-like\" with the suffix -virales indicating a virus order.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row65\" class=\"row_heading level0 row65\" >65</th>\n",
              "      <td id=\"T_b4c57_row65_col0\" class=\"data row65 col0\" >Viral replication</td>\n",
              "      <td id=\"T_b4c57_row65_col1\" class=\"data row65 col1\" >Viral replication is the formation of biological viruses during the infection process in the target host cells.  Viruses must first get into the cell before viral replication can occur.  Through the generation of abundant copies of its genome and packaging these copies, the virus continues infecting new hosts.  Replication between viruses is greatly varied and depends on the type of genes involved in them. Most DNA viruses assemble in the nucleus while most RNA viruses develop solely in cytoplasm.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row66\" class=\"row_heading level0 row66\" >66</th>\n",
              "      <td id=\"T_b4c57_row66_col0\" class=\"data row66 col0\" >Viriome</td>\n",
              "      <td id=\"T_b4c57_row66_col1\" class=\"data row66 col1\" >The viriome of a habitat or environment is the total virus content within it. A viriome may relate to the viruses that inhabit a multicellular organism as well as the phages that are residing inside bacteria and archaea.\n",
              "This term exists in contrast to the virome, which more commonly refers to the collection of nucleic acids contained by viruses in a microbiome.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row67\" class=\"row_heading level0 row67\" >67</th>\n",
              "      <td id=\"T_b4c57_row67_col0\" class=\"data row67 col0\" >Virology</td>\n",
              "      <td id=\"T_b4c57_row67_col1\" class=\"data row67 col1\" >Virology is the scientific study of viruses – submicroscopic, parasitic organisms of genetic material contained in a protein coat – and virus-like agents. It focuses on the following aspects of viruses: their structure, classification and evolution, their ways to infect and exploit host cells for reproduction, their interaction with host organism physiology and immunity, the diseases they cause, the techniques to isolate and culture them, and their use in research and therapy. Virology is a subfield of microbiology.\n",
              "The identification of the causative agent of tobacco mosaic disease (TMV) as a novel pathogen by Martinus Beijerinck (1898) is now acknowledged as being the official beginning of the field of virology as a discipline distinct from bacteriology. He realized the source was neither a bacterial nor a fungal infection, but something completely different. Beijerinck used the word ‘virus’ to describe the mysterious agent in his ‘contagium vivum fluidum’ (‘contagious living fluid’).  Rosalind Franklin proposed the full structure of the tobacco mosaic virus in 1955.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row68\" class=\"row_heading level0 row68\" >68</th>\n",
              "      <td id=\"T_b4c57_row68_col0\" class=\"data row68 col0\" >Virosphere</td>\n",
              "      <td id=\"T_b4c57_row68_col1\" class=\"data row68 col1\" >Virosphere (virus diversity, virus world, global virosphere) is viral part of the biosphere, namely the pool of viruses in all hosts and all environments on planet earth.Virosphere may also refer to a pool of certain group of viruses according to their host - prokaryotic virosphere, archaeal virosphere,  Invertebrate  virosphere,  type of genome  - RNA virosphere,  dsDNA virosphere or ecological niche - marine virosphere</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th id=\"T_b4c57_level0_row69\" class=\"row_heading level0 row69\" >69</th>\n",
              "      <td id=\"T_b4c57_row69_col0\" class=\"data row69 col0\" >Social history of viruses</td>\n",
              "      <td id=\"T_b4c57_row69_col1\" class=\"data row69 col1\" >The social history of viruses describes the influence of viruses and viral infections on human history. Epidemics caused by viruses began when human behaviour changed during the Neolithic period, around 12,000 years ago, when humans developed more densely populated agricultural communities. This allowed viruses to spread rapidly and subsequently to become endemic. Viruses of plants and livestock also increased, and as humans became dependent on agriculture and farming, diseases such as potyviruses of potatoes and rinderpest of cattle had devastating consequences.\n",
              "Smallpox and measles viruses are among the oldest that infect humans. Having evolved from viruses that infected other animals, they first appeared in humans in  Europe and  North Africa thousands of years ago. The viruses were later carried to the New World by Europeans during the time of the Spanish Conquests, but the indigenous people had no natural resistance to the viruses and millions of them died during epidemics. Influenza pandemics have been recorded since 1580, and they have occurred with increasing frequency in subsequent centuries. The pandemic of 1918–19, in which 40–50 million died in less than a year, was one of the most devastating in history.\n",
              "Louis Pasteur and Edward Jenner were the first to develop vaccines to protect against viral infections. The nature of viruses remained unknown until the invention of the electron microscope in the 1930s, when the science of virology gained momentum. In the 20th century many diseases both old and new were found to be caused by viruses. There were epidemics of poliomyelitis that were only controlled following the development of a vaccine in the 1950s. HIV is one of the most pathogenic new viruses to have emerged in centuries. Although scientific interest in them arose because of the diseases they cause, most viruses are beneficial. They drive evolution by transferring genes across species, play important roles in ecosystems and are essential to life.</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n"
            ]
          },
          "metadata": {},
          "execution_count": 21
        }
      ]
    }
  ]
}